pkg/cluster/cluster.go

```diff
@@ -479,6 +479,15 @@ func (c *Cluster) pollPods() ([]*v1.Pod, []*v1.Pod, error) {
 }
 
 func (c *Cluster) updateStatus() error {
+	if c.bm != nil {
+		bs, err := c.bm.getStatus()
```

Comment:
IMO, getStatus() should not be put in updateStatus(). There are two cases it will surely fail:
- Update status from None -> Creating
- Update status from Creating -> Running
- When setup() failed
- Initial rounds of reconcile(). Backup pod could take a while to start due to PV mount.

Although we can tolerate the last point, it didn't make sense to add the overhead to other points.